ovos_number_parser/__init__.py

```
@@ -50,6 +49,8 @@ def numbers_to_digits(utterance: str, lang: str, scale: Scale = Scale.LONG) -> s
     """
     if lang.startswith("az"):
         return numbers_to_digits_az(utterance)
+    if lang.startswith("ast"):
+        return numbers_to_digits_ast(utterance)
```

Ruff (0.14.7)

Undefined name `numbers_to_digits_ast`

(F821)